app/src/main/java/com/tma/sparking/services/parkinghandler/ParkingDataCallback.java

```diff
@@ -0,0 +1,9 @@
+package com.tma.sparking.services.parkinghandler;
+
+import com.tma.sparking.models.ParkingField;
+
+import java.util.List;
+
+public interface ParkingDataCallback {
```

Comment:
When using ContentProvider for saving data, this interface is not required anymore. The activity can communicate directly to ContentProvider using CursorLoader.